bin/offline-dbg

```diff
@@ -162,19 +162,40 @@ do_collect-sos() {
     local sos=$1
 
     local dir_name=`tar -tf $sos | head -1 | cut -f1 -d"/"`
-    sos_files="$dir_name/sos_commands/rpm
-    $dir_name/sos_commands/openvswitch
-    $dir_name/var/lib/openvswitch
-    $dir_name/etc/openvswitch
-    $dir_name/usr/local/etc/openvswitch"
+
+    sos_files="${dir_name}/sos_commands/rpm
+    ${dir_name}/sos_commands/openvswitch"
+
+    db_locations="var/lib/openvswitch
+    etc/openvswitch
+    usr/local/etc/openvswitch"
 
     if test -f "$sos"; then
         mkdir ${WORKDIR}/sos_report
         echo "Extracting sos report from '$sos'..."
-        tar -xvf $sos -C ${WORKDIR}/sos_report $sos_files --strip-components 1 --ignore-failed-read &>/dev/null || echo "Error occured while extracting specified \$sos_files from $sos"
-        CMD_DIR=${WORKDIR}/sos_report/sos_commands/openvswitch/
     else
-        error "SOS archive file not found. Please specify a <sosreport>.tar.xz archive"
+        echo "SOS archive file not found. Please specify a <sosreport>.tar.xz archive"
+    fi
+
+    for dir in $sos_files; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report $dir --strip-components 1 &>/dev/null || error "Could not extract critical directory $dir from $sos"
+    done
+    CMD_DIR=${WORKDIR}/sos_report/sos_commands/openvswitch/
+
+    for dir in $db_locations; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report ${dir_name}/${dir}/conf.db --strip-components 1 &>/dev/null || true
+        if test -f "${WORKDIR}/sos_report/${dir}/conf.db"; then
+            db_dir=$dir
```

Comment:
don't use a global variable for this. declare it local outside of the loop please